aws/data_source_aws_db_instance_test.go

```diff
@@ -42,8 +42,11 @@ func TestAccAWSDbInstanceDataSource_basic(t *testing.T) {
 }
 
 func TestAccAWSDbInstanceDataSource_ec2Classic(t *testing.T) {
+	hardcodedRegion := "us-east-1" // lintignore:AWSAT003
+	testAccReachableRegionPreCheck(hardcodedRegion, t)
+
 	oldvar := os.Getenv("AWS_DEFAULT_REGION")
-	os.Setenv("AWS_DEFAULT_REGION", "us-east-1")
+	os.Setenv("AWS_DEFAULT_REGION", hardcodedRegion)
 	defer os.Setenv("AWS_DEFAULT_REGION", oldvar)
```

Comment:
`us-east-1` is not the only region that supports EC2-Classic, and it is even supported in GovCloud. I've written up #15791, which could be used here and in other EC2-Classic-related tests

Reply:
Deferring for another PR